src/mongoc/mongoc-client.c

```diff
@@ -92,13 +90,8 @@ mongoc_client_connect_tcp (const mongoc_uri_t       *uri,
    bson_return_val_if_fail (uri, NULL);
    bson_return_val_if_fail (host, NULL);
 
-   if ((options = mongoc_uri_get_options (uri)) &&
-       bson_iter_init_find_case (&iter, options, "connecttimeoutms") &&
-       BSON_ITER_HOLDS_INT32 (&iter)) {
-      if (!(connecttimeoutms = bson_iter_int32(&iter))) {
-         connecttimeoutms = MONGOC_DEFAULT_CONNECTTIMEOUTMS;
-      }
-   }
+   connecttimeoutms = mongoc_uri_get_option_as_int32(uri, "connecttimeoutms",
+         MONGOC_DEFAULT_CONNECTTIMEOUTMS);
```

Comment:
Let's keep to a style more like this, where args are left-aligned and lines are less than 80 chars:
```
   connecttimeoutms = mongoc_uri_get_option_as_int32(
      uri, "connecttimeoutms",
      MONGOC_DEFAULT_CONNECTTIMEOUTMS);
```